### ISSUE TITLE
get-schema discards custom directives?

### ISSUE THREAD

**Issue by mdecurtins** [Urigo/graphql-cli#536]
I'm trying to generate a SDL dump of my schema, and it appears that the generated SDL ignored all my custom schema directives and fields I annotated with those directives. `get-schema` appears to keep directives defined by the GraphQL spec, e.g. `@deprecated`, but discards custom directives.

Can anyone confirm this is a current limitation of `get-schema`? Is there any way to get my directives into the generated SDL?

What I did:
```
npm install -g graphql-cli
graphql get-schema --endpoint http://localhost:4000/graphql --output schema.graphql
```

Using:
- `graphql-cli`: 3.0.14
- `apollo-server-express`: 2.4.8
- `graphql`: 14.2.1

**EDIT:** I'm aware that introspection doesn't pick up custom directives (https://github.com/graphql/graphql-spec/issues/300 and https://github.com/graphql/graphql-js/issues/1072) but I'm wondering if Apollo has a way to get at them?

**Comment by ardatan** [comment]
Currently we have new structure for GraphQL CLI and in the new alpha version `4.0.0-alpha.0` we suggest to use GraphQL Code Generator plugin of GraphQL CLI with Codegen's `schema-ast` plugin. Checkout new instructions and we will be happy to hear your feedback. Feel free to contact us using our Discord channel;
https://discord.gg/xud7bH

**Comment by Urigo** [comment]
We've just released a new alpha version channel for GraphQL CLI - `4.0.0-alpha.XXX`!

We've changed the `get-schema` command to use [GraphQL Code Generator](http://graphql-code-generator.com) plugin of GraphQL CLI with the Codegen's [`schema-ast` plugin](https://graphql-code-generator.com/docs/plugins/schema-ast) which can handle directives.

Checkout new instructions in the [migration guide](https://github.com/urigo/graphql-cli#migration-from-3xx-to-4xx) and let us know your feedback during that alpha period!

Feel free to contact us by opening a new issue or joining our Discord channel;
https://discord.gg/xud7bH
